### ISSUE TITLE
[1.12.2] ME Drive Fixture disconnects and crashes

### ISSUE THREAD

**Issue by ZeekDaGeek** [ExtraCells/ExtraCells2#516]
MC: 1.12.2
Mod: ExtraCells-1.12.2-2.5.10a47.jar
Forge: 14.23.1.2582
Mod pack: Modern Skyblock 2 v2.8.1.1

For some reason it seems that ME Drive Fixtures are sending data about what they have in them when you look into them resulting in issues if your ME Drives are filled with very complicated items. In this case I have a 1k ME Drive filled with Forestry Bees (massive amounts of extra NBT info).

![Complex Drive](http://zeekdageek.tv/ss/MORzD.png)

Every time I open a ME Drive Fixture with one of these in it will kick me from the server saying `Internal Exception: java.lang.IllegalArugmentException: Paylout may not be larger than 32767 bytes.` This same thing won't happen if you use a standard ME Drive though.

Additionally ME Drive Fixtures seem to have issues rendering and will cause client crashed when looked at from a specific direction: 

![ME Drive Fixture Position](http://zeekdageek.tv/ss/x4fd6.png)
If an ME Drive Fixture is located in the position of the ME Drive with the torch on it my client will crash every time with the following crash log: https://pastebin.com/BJFqNPkV

The position is specifically 11 blocks forward and 2 blocks to the left, here's an arial view for distance reference, the player head being where I would stand with a nice dirt body:

![Crash distance](http://zeekdageek.tv/ss/8m9oj.png)

**Comment by Rixxaw** [comment]
I'm also having this issue, I imported my chests and now I can't get my drives out of my fixture cuz it just crashes each time i try to open the fixture.  its only 1 fixture, I'll move it to a sub network and see if I can pull out the high NBT data items and hopefully fix it.

**Comment by Rixxaw** [comment]
Yep i just got the payload error. exact same issue... I will try to find the item inside that is causing it

**Comment by LastXsile** [comment]
Just had this happen just now and had to rollback the server, makes them very dangerous to use in big mod packs with very complicated items.

**Comment by Dandycarrot** [comment]
A temporary fix for those wishing to retrieve their drives from the fixture is to empty the items from the drives in the fixture. You should then be able to remove the drives without crashing.
